### PR TITLE
A lot of formatting fixes

### PR DESCRIPTION
**Pull request by steven-murray**
Mostly whitespace, but also trying to reduce code complexity

### CODE REVIEW

File: cal_coefficients.py [hunk 1]
```diff
@@ -6,40 +6,37 @@
...
 import os
-clear = lambda : os.system('cls')
+
+clear = lambda: os.system('cls')
 clear()
-import numpy as np
-import scipy.io as sio
 from receiver_calibration_func import *
 import glob
-import reflection_coefficient as rc
```
Comment (dmlewis9) [inline cal_coefficients.py]: Needed starting on line 290

Comment (steven-murray) [reply cal_coefficients.py]: Good catch, fixed in a4dbf9d4032ab5fe7c899c0395ff057b0e7768a0

File: cal_coefficients.py [hunk 2]
```diff
@@ -6,40 +6,37 @@
...
+clear = lambda: os.system('cls')
 clear()
-import numpy as np
-import scipy.io as sio
-from receiver_calibration_func import *
+import receiver_calibration_func as rcf
 import glob
-import reflection_coefficient as rc
-from S11_correction import *
+import S11_correction as s11
```
Comment (dmlewis9) [inline cal_coefficients.py]: calling this as s11 is generating an error. Line 442 is using 's11' as a variable, one or the other will have to change in the next commit.

Comment (steven-murray) [reply cal_coefficients.py]: Yup fair point, fixed in 30d389d5952adaa9097c7b9c5876f35d8818b74c

**Comment by steven-murray** [comment]
OK I just pushed a reasonably big change that modularised a lot of the copy/paste stuff in `cal_coefficients`. It runs through for me without raising an exception, however it _is_ reporting 

```
/data4/smurray/Projects/radio/EOR/Edges/cal_coefficients/calibrate/modelling.py:244: LinAlgWarning: Ill-conditioned matrix (rcond=1.13415e-17): result may not be accurate.
  param = sp.linalg.solve(R1, np.dot(Q1.T, Wydata))
```

So I'm assuming that somewhere along the line I got something wrong (maybe using the wrong variable somewhere). I don't know how to interpret the plots etc., so it's hard for me to locate where the error might be. 

There's a new script in `devel/` that basically does what your notebook did -- you will have to slightly update the notebook to be in line with this script.

**Comment by steven-murray** [comment]
Are we ready to pull this in?

**Comment by dmlewis9** [comment]
Should be good for pulling now.

**Pull request merged**